### PR TITLE
Clone item

### PR DESCRIPTION
**Pull request by rmisio**
https://github.com/OpenBazaar/OpenBazaar-Client/issues/394

### CODE REVIEW

**Comment by rmisio** [comment]
Found a bug. Will fix and re-open PR.

**Comment by rmisio** [comment]
Ok, ready to go.

File: js/views/itemEditVw.js [hunk 1]
```diff
@@ -55,7 +56,16 @@ module.exports = Backbone.View.extend({
     self.model.set("imageHashesToUpload", anotherHashArray);
     self.model.set('expTime', self.model.get('vendor_offer').listing.metadata.expiry.replace(" UTC", ""));
 
-    this.render();
+    // prevent the body from picking up drag actions
+    $(document.body).on("dragover", this.onDragonover = function(e) {
+      e.preventDefault();
+      return false;
+    });
```
Comment (jjeffryes) [inline js/views/itemEditVw.js:63]: Isn't this already being done when the app starts up, or did it accidentally get removed?

Comment (jjeffryes) [inline js/views/itemEditVw.js:63]: edit: I see this was being done in render. It should be ok to remove it entirely.

**Pull request merged**